### ISSUE TITLE
Bug: LGU Save Files not able to sync with LCBetterSaves

### ISSUE THREAD

**Issue by Uncryptical** [Malcolm-Q/LC-LateGameUpgrades#15]
I've noticed that the LGU save files are not syncing with the save files when using LCBetterSaves.

It's not actually that big of an issue, but it has to actually do with the way the actual Game save files and your LGU save files are named. Your LGU save files work perfectly fine EXCEPT for the fact that they aren't matching up with certain save files when one gets deleted.
So for example: the creator of LCBetterSaves has made his save files renamed whenever a save file gets deleted.

**My personal example is here:** LCSaveFiles 0-4, and your LGU save files 3 and 4. We'll only look at 3 and 4 specifically to recreate this problem. LGU_3 has the Running Shoes Upgrade, while LGU_4 does not. (Ignore everything past LGU_4.json)
![LGU1](https://github.com/Malcolm-Q/LC-LateGameUpgrades/assets/55523837/bfe84562-cf35-44fc-a675-7f142c1b1b6a)

Now, if I go into LC and delete LCSaveFile3 (AKA the 4th Save Slot), all the files AFTER the original LCSaveFile3 get their save number subtracted by 1. The Original LCSaveFile3 = Gone, The Original LCSaveFile4 = The New LCSaveFile3, etc.
![LGU2](https://github.com/Malcolm-Q/LC-LateGameUpgrades/assets/55523837/aadd0701-cf2b-475e-9e41-9eb28762f4aa)

NOW, the original LGU_4 is not being applied to the New LCSaveFile3. The original LGU_3 is now being applied to the New LCSaveFile3.

***In simple terms: when I delete the 1st Save slot, the 2nd Save slot becomes the new 1st Save. BUT, now the new 1st Save (2nd Save) has the original 1st Upgrade Save, which is being caused by the names of the save files. Oh, and the original 2nd Upgrade Save is just kinda there.***

**Comment by Malcolm-Q** [comment]
I see it has a repository on GitHub as well. I can look into this and see if the owner would be receptive to me making a PR changing the delete method of their mod to be a safer operation. Thanks.

**Comment by Malcolm-Q** [comment]
@Uncryptical Paul merged my pull request a few days ago and said he will update it on thunderstore soon here.

**Comment by paul-merkamp** [comment]
should be updated in thunderstore now, LMK if there are further issues
